entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+echo "Making migrations"
+python manage.py migrate
+
+echo "Creationg superuser"
+python manage.py createsuperuser --noinput
```

Comment:
Not the best way to create superuser, it will create instance on each start, moreover I don't see where creds are used